external/st_audiofile/src/st_audiofile.h

```diff
@@ -19,6 +19,8 @@
 #include <wchar.h>
 #endif
 
+#define WAVPACK_MEMORY_ASSUMED_VERSION 5
```

Comment:
Does this need to be in the header file?

Reply:
No reason to have in the header file, https://github.com/sfztools/sfizz/pull/1129/commits/c3c416496f26b41b27c82d8e1adf750e4e24ed27 looks good.